### PR TITLE
Initial icon themes

### PR DESCRIPTION
**Pull request by joebonrichie**

### CODE REVIEW

File: a/adwaita-icon-theme/manifest.x86_64.jsonc [hunk 1]
```diff
@@ -0,0 +1,927 @@
...
+			"build-depends": [
+				"binary(gtk-update-icon-cache)"
+			],
+			"files": [
+				"/usr/share/pkgconfig/adwaita-icon-theme.pc"
+			],
+			"name": "adwaita-icon-theme-devel",
+			"provides": [
+				"pkgconfig(adwaita-icon-theme)"
+			]
```
Comment (der-eismann) [inline a/adwaita-icon-theme/manifest.x86_64.jsonc:921]: Should this go into the main package? Not sure if we need a separate package just for the pkgconfig

File: a/adwaita-icon-theme/stone.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#
+# SPDX-FileCopyrightText: © 2020-2023 Serpent OS Developers
+#
+# SPDX-License-Identifier: Zlib
+#
```
Comment (der-eismann) [inline a/adwaita-icon-theme/stone.yml:5]: Wrong (old) header in both files. I think it was updated in boulder recently.

**Review by der-eismann — COMMENTED** [review]
Could you please add the monitoring.yaml as well?
https://release-monitoring.org/project/1316/
https://release-monitoring.org/project/13117/

**Comment by der-eismann** [comment]
LGTM from my side! :slightly_smiling_face: Waiting for big boss though since it will increase package base.

**Comment by ermo** [comment]
@joebonrichie Add triggers for the icon caches please.

Based on a discussion I just had with Ikey, I'm pretty sure we can make caching work for the icon caches thing in the future, so let's put some pressure on ourselves to get caching done by adding the triggers now.

**Comment by ikeycode** [comment]
We can add triggers to one of the gtk packages, I just need to figure out which one and minimise the deps. Don't consider a blocker !

**Review by ikeycode — APPROVED** [review]

**Pull request merged**